pkg/tests/stamp_test.py

```diff
@@ -13,17 +13,52 @@
 # limitations under the License.
 """Test time stamping in pkg_tar"""
 
+import datetime
 import tarfile
 import time
 import unittest
+import zipfile
 
 from bazel_tools.tools.python.runfiles import runfiles
 
 # keep in sync with archive.py
 PORTABLE_MTIME = 946684800  # 2000-01-01 00:00:00.000 UTC
 
-class PkgTarTest(unittest.TestCase):
-  """Testing for pkg_tar rule."""
+
+class StampTest(unittest.TestCase):
+  """Test for time stamps in packages."""
```

Comment:
This implies this might also test the normal case where artifacts have a consistent time.  This doesn't seem to happen here.

It might be worth clarifying this point, and perhaps also stating somewhere the test for the default epoch is elsewhere.

Reply:
Right.  I changed the comment to indicate non-epochness.